lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -103,9 +103,6 @@ class AxeAudit extends Audit {
 
     return {
       score: Number(rule === undefined),
-      extendedInfo: {
-        value: rule,
```

Comment:
we don't expose the full axe rule output otherwise.

but i think that's okay

here's a full "rule":

![image](https://user-images.githubusercontent.com/39191/82086652-af1d7380-96a3-11ea-8ef8-2596115db0b8.png)

we'd miss out on their description & title and url.
and for each node we'd miss out on its specific `impact` and `failureSummary`


i think this is ok.


somewhat related: `failureSummary` tbh looks useful for our regular output. requires a bit more investigation.

Reply:
Should I add a note or TODO here?

Reply:
> and for each node we'd miss out on its specific `impact` and `failureSummary`

`failureSummary` becomes each node's `explanation` in the axe audit tables, e.g.

https://github.com/GoogleChrome/lighthouse/blob/e2bfa1450b310f2441122e7f8a16c7c9def63144/lighthouse-core/test/results/sample_v2.json#L1965-L1972

Reply:
(after #5402 asked us to put stuff back)

Reply:
Should I add a TODO or something to remind us to come back to this?

Reply:
> Should I add a TODO or something to remind us to come back to this?

we've been happy with the current output since #5402, so how about let's leave as it is here and re-visit if someone asks for more or we have ideas for future reporting that needs more.